<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Relatorio</class>
 <widget class="QWidget" name="Relatorio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbCadastrar">
       <property name="text">
        <string>Cadastrar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbRelatorio">
       <property name="text">
        <string>Relatorio</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>Filial</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Produto</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Categoria</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Base</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
